statefun-docs/src/main/java/org/apache/flink/statefun/docs/io/kafka/IngressSpecs.java

```diff
@@ -31,8 +31,9 @@
   public static final IngressSpec<User> kafkaIngress =
       KafkaIngressBuilder.forIdentifier(ID)
           .withKafkaAddress("localhost:9092")
+          .withConsumerGroupId("greetings")
```

Comment:
I wondering rather we should set the withStartupPosition() in .withConsumerGroupId() in the ingress builder.